src/gui/sdlmain.cpp

```diff
@@ -485,9 +485,10 @@ static void populate_requested_vsync_settings()
 		sdl.vsync.when_fullscreen.requested = VsyncState::Yield;
 	} else {
 		assert(user_pref == "auto");
-		// In windowed-mode, assume the window manager has exclusive
-		// access to the GPU so leave vsync off
-		sdl.vsync.when_windowed.requested = VsyncState::Off;
+		// In windowed-mode, assume modern compositors are using some
+		// form of vsync or frame limiting to prevent visual tearing of
+		// the desktop itself.
+		sdl.vsync.when_windowed.requested = VsyncState::On;
```

Comment:
Yes, the comment is correct and that's the case, but that means we don't need to enable vsync ourselves in windowed mode.

We just send frames whenever in windowed mode, and it will be vsync'ed by the compositor of the OS, as if "by magic".

That's my experience on Windows with other emulators; you absolutely don't _need_ to manually enable vsync in windowed mode. Well, probably it's a no-op anyway, but I'll leave this off for clarity.

Reply:
We can leave vsync off in windowed mode, but I still need (or strongly prefer) being able to make the reasonable assumption that we're "effectively-vsyncd", or rate-bound no faster than the reported display's refresh rate when both in windowed and when the vsync setting is auto.

Reply:
Ok, in that case the error is mine 😄 I thought this means you're trying to _enforce_ vsync through SDL, or by setting `glSwapInterval(1)` or something. If this is merely an assumption about what is _already happening_ in windowed mode, then yes, this is actually correct and I take it all back 😄 

Haven't read the whole code, maybe I should've...

Reply:
Updated autoto the assume vsync, but not set it.